src/kibana/styles/_variables.less

```diff
@@ -20,3 +20,5 @@
 
 @action-add: desaturate(@btn-success-bg, 30%);
 @action-remove: desaturate(@btn-danger-bg, 30%);
+
+@agg-table-row-hover: lighten(@gray-lighter, 4%);
```

Comment:
I think it might make sense to keep all of these changes in agg_table.less, and not create a new variable here, because I'm not sure it will be used anywhere else.

I guess I'm just suggesting replacing `@agg-table-row-hover` inside agg_table.less with `lighten(@gray-lighter, 4%);` and removing this new variable from this file.